core/src/test/scala/org/apache/spark/deploy/worker/WorkerSuite.scala

```diff
@@ -51,7 +51,7 @@ class WorkerSuite extends SparkFunSuite with Matchers with BeforeAndAfter {
 
   import org.apache.spark.deploy.DeployTestUtils._
 
-  @Mock(answer = RETURNS_SMART_NULLS) private var shuffleService: ExternalShuffleService = _
+  @Mock(answer = RETURNS_SMART_NULLS) private val shuffleService: ExternalShuffleService = null
```

Comment:
Hm, I was going to wonder if we have to leave mocked members as 'var' because the framework will change them. But it looks like tests pass.

Reply:
https://github.com/mockito/mockito/blob/019830665f97c8ba6159148e3b10086cfb894900/src/main/java/org/mockito/internal/util/reflection/FieldSetter.java#L13-L18

Reply:
Sure, maybe revert those just to be conservative.

Reply:
done ~